agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -4580,6 +4579,41 @@ bool backhaul_manager::handle_backhaul_steering_request(ieee1905_1::CmduMessageR
     return send_cmdu_to_bus(cmdu_tx, controller_bridge_mac, bridge_info.mac);
 }
 
+bool backhaul_manager::create_backhaul_steering_response(
+    const wfa_map::tlvBackhaulSteeringResponse::eResultCode &result_code)
+{
+    auto cmdu_tx_header =
+        cmdu_tx.create(0, ieee1905_1::eMessageType::BACKHAUL_STEERING_RESPONSE_MESSAGE);
```

Comment:
Is it OK to use message ID 0?
Often message ID of the request and the response should be the same.
Certification test 4.9.1 does not check it. Can it change in the future?